astropy_timeseries/io/kepler.py

```diff
@@ -1,37 +1,98 @@
-from astropy.io import registry
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import warnings
+
+import numpy as np
+
+from astropy.io import registry, fits
 from astropy.table import Table
-from astropy.time import Time
+from astropy.time import Time, TimeDelta
 
-from ..sampled import TimeSeries
+from astropy_timeseries.sampled import TimeSeries
 
-__all__ = ['kepler_fits_reader']
+__all__ = ["kepler_read"]
 
 
-def kepler_fits_reader(filename):
+def kepler_read(filename):
+    """
+    This serves as the FITS reader for KEPLER or TESS files within astropy-timeseries.
 
-    # Parse Kepler FITS file with regular FITS reader
-    tab = Table.read(filename, format='fits')
+    This allows reading a supported FITS file using syntax such as::
+    >>> from astropy_timeseries.io import kepler
+    >>> timeseries = kepler.kepler_reader('<name of fits file>')  # doctest: +SKIP
```

Comment:
This shouldn't be the recommended way, we should keep ``TimeSeries.read(..., format='kepler.fits')`` as the recommended API.